mentat-vscode/src/utils/server.ts

```diff
@@ -9,6 +9,10 @@ import { StreamMessage } from "types";
 import * as util from "util";
 import { v4 } from "uuid";
 import * as vscode from "vscode";
+
+// IMPORTANT: This MUST be updated with the vscode extension to ensure that the right mentat version is installed!
+const MENTAT_VERSION = "1.0.10";
```

Comment:
Oh hmm that would be a better idea to grab it from mentat/metnat/version; i'll do that. The reason we can't pip --upgrade is because I don't want to have to worry about backwards compatibility; this way, if somebody is using an older version of the extension it will automatically install the compatible version of mentat.

Reply:
I think I'll add a build step that injects the version from mentat/mentat/version into the extension in another PR. I haven't messed with the build steps too much yet and it isn't really necessary right now.